apps/prairielearn/src/server.js

```diff
@@ -1312,6 +1312,14 @@ export async function initExpress() {
     },
     (await import('./pages/instructorQuestions/instructorQuestions.js')).default,
   ]);
+  app.use('/pl/course_instance/:course_instance_id(\\d+)/instructor/ai-generate-question', [
```

Comment:
We'll also need to fix up anything that calls these endpoints.
```suggestion
  app.use('/pl/course_instance/:course_instance_id(\\d+)/instructor/ai_generate_question', [
```